templates/sshd.custom.erb

```diff
@@ -0,0 +1,15 @@
+# This file is being maintained by Puppet.
```

Comment:
Took the route with a variable as suggested by @Phil-Friderici , let me know what you think